testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionServiceBundleIT.java

```diff
@@ -760,4 +762,22 @@ public void testDeleteOldCompletedSubscription() {
         subscriber = subscriberDataService.findByCallingNumber(2000000000L);
         assertEquals(1, subscriber.getSubscriptions().size());
     }
+    
+    @Ignore
+    @Test
+    public void verifyFT153() {
+    	/*
```

Comment:
@AnkitGaurNms @PriyaJain5 @kanchannms @srivastavaAbhi Please make sure to add the comment for the method on top and not inside. I understand @frankhuster might have started this trend originally, but we should stick with the convention and move the comments to the top of the method.